.github/workflows/externalworkloads.yaml

```diff
@@ -87,21 +106,42 @@ jobs:
         run: |
           gcloud compute ssh ${{ env.vmName }} --zone ${{ env.zone }} --command "ping -c 3 \$(cilium service list get -o jsonpath='{[?(@.spec.flags.name==\"clustermesh-apiserver\")].spec.backend-addresses[0].ip}')"
 
-      - name: Connectivity test
+      - name: Load cilium test script in configmap
         run: |
-          cilium connectivity test --all-flows
+          kubectl create configmap cilium-cli-test-script -n kube-system --from-file=in-cluster-test-script.sh=.github/in-cluster-test-scripts/external-workloads.sh
 
-      - name: Cleanup
+      - name: Create cilium-cli test job
+        run: |
+          helm install .github/cilium-cli-test-job-chart --generate-name \
+          --set tag=${{ steps.vars.outputs.sha }} \
+          --set job_name=cilium-cli \
+          --set test_script_cm=cilium-cli-test-script \
+
+      - name: Wait for test job
+        run: |
+          kubectl -n kube-system wait job/cilium-cli --for=condition=complete --timeout=10m
```

Comment:
Doesn't this completely hide the test output in case of failure?

It used to fail with the output of cilium connectivity test clearly detailed what failed. Now it just says a timeout was reached...